src/theory/bv/theory_bv_rewriter.cpp

```diff
@@ -97,6 +97,45 @@ Node TheoryBVRewriter::rewriteViaRule(ProofRewriteRule id, const Node& n)
   return Node::null();
 }
 
+TrustNode TheoryBVRewriter::expandDefinition(Node node)
+{
+  Node expanded = eliminateOverflows(node);
+  return TrustNode::mkTrustRewrite(node, expanded, nullptr);
```

Comment:
Reading the expandDefinition documentation, I'm not sure if expandDefinition is the right way to do this since these eliminations can be easily done via rewrites.

Reply:
Similarly to the above, the rewriter eliminates them already in apply-substs, which is why we want to dealy their elimination.